sensor_msgs/PointCloud2 frenet_path
char frenet_state

